scss/_carousel.scss

```diff
@@ -210,7 +210,7 @@
 
 // Dark mode carousel
 
-.carousel-dark {
+%carousel-dark {
   .carousel-control-prev-icon,
```

Comment:
On dark mode, it is very difficult to distinguish the normal color and the hovered one.

Reply:
Noted, let's save for later with more feedback.